libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -342,7 +342,12 @@ def add_version_header(tc):
         },
         {
             "name": "__cpp_lib_constexpr_complex",
-            "values": {"c++20": 201711},
+            "values": {"c++26": 201711},
+            "headers": ["deque"],
+        },
+        {
+            "name": "__cpp_lib_constexpr_deque",
+            "values": {"c++20": 202502},
```

Comment:
Current changes are incorrect. The entry for `__cpp_lib_constexpr_complex` should be kept unchanged, while the `"value"` field of `__cpp_lib_constexpr_deque` should be `{"c++26": 202502}`.